gulpfile.js

```diff
@@ -167,12 +171,26 @@ gulp.task('copy:safari', ['clean:safari', 'lint'], function() {
 
 });
 
+gulp.task('copy:edge', ['clean:edge', 'lint'], function() {
+	var src = [].concat(
+		paths.scripts.core,
+		paths.css.core,
+		paths.vendor,
+		['Chrome/bridge.js', 'Chrome/icon.png', 'Edge/**/*']
```

Comment:
probably should copy icon to Edge or a shared directory (webkit?) (also see my comments about bridge above)